module/zstd/README.md

```diff
@@ -10,9 +10,8 @@ library, besides upgrading to a newer ZSTD release.
 Tree structure:
 
 * `zfs_zstd.c` is the actual `zzstd` kernel module.
-* `lib/` contains the unmodified, [_"amalgamated"_](https://github.com/facebook/zstd/blob/dev/contrib/single_file_libs/README.md)
-  version of the `Zstandard` library, generated from our template file
-* `zstd-in.c` is our template file for generating the library
+* `lib/` contains the unmodified version of the `Zstandard` library
```

Comment:
I *am* being pedantic here but since you're already touching it, the library is Zstandard *or* `zstd`, but not `Zstandard`.